### DISCUSSION TITLE
Chrome://flags export???

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#1405] [Q&A] [answered]
Can experimental setting exported? 
@Eloston

**Comment by networkException** [comment, marked as answer]
Flags are stored in a json file called `Local State` inside the [user data directory](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/user_data_dir.md). Depending on your operating system you might also be able to look at the command arguments of the chromium process and extract them into a start script, desktop file or similar

**Reply by ghost** [reply]
bruh me on android not a pc is not possible to backup those setting tho?

**Reply by networkException** [reply]
Maybe the `Local State` file also exists in the user data directory on android, I don't know
